src/frameobject.ts

```diff
@@ -66,7 +68,16 @@ class Py_FrameObject {
         this.outputDevice = outputDevice;
         this.shouldWriteSpace = false;
         this.blockStack = [];
-        this.env = env;
+        this.env = [];
+        
+        for (var i = 0; i < code.cellvars.length; i++) {
+            this.env.push(new Py_Cell(null));
+        }
+
+        for (var i = 0; i < code.freevars.length; i++) {
```

Comment:
Eventually, I think I'm going to switch on linter options to prevent multiple definitions of the same variable. It's innocuous here, but since JavaScript has function scope, these two `i` variables are technically the same variable.

When I write code, you'll often see me declare all of my variables w/ explicit types at the top of the scope. It's a JavaScript habit I've developed.